04-gas/gas-section-two/gas-section-two.html

```diff
@@ -16,20 +16,22 @@
 
       <section class="gas-section-two-subcontainer">
 
-        <div class="gas-sectin-two-title-container">
-          <h2 class="gas-section-two-subtitle">2</h2>
-          <h1 class="gas-section-two-title">
-            Only turn on the heating when you need it
-          </h1>
+        <div class="gas-section-two-copy-container">
+
+          <div class="gas-section-two-title-container">
+            <h2 class="gas-section-two-title">
+             <span class="gas-section-two-span-number">2. </span><span class="gas-section-two-span-title"> Only <span class="bold">turn on the heating</span> when you<span class="bold"> need it</span></span>
```

Comment:
maybe something like, because we want to identify the parent really.. and then the sub element and then its purpose
`class="gas-section-two-title-span-number"`
`class="gas-section-two-title-span-title"`